Library/Homebrew/dev-cmd/sh.rb

```diff
@@ -10,18 +10,19 @@ module Homebrew
   def sh_args
     Homebrew::CLI::Parser.new do
       usage_banner <<~EOS
-        `sh` [<options>]
+        `sh` [<options>] [<commands>]
```

Comment:
```suggestion
        `sh` [<options>] [<script>]
```

Because it's only taking a script as named arguments now, right?

Reply:
Oh, right. I need to make this change manually to update manual pages